aviary/utils/process_input_decks.py

```diff
@@ -183,7 +189,7 @@ def update_GASP_options(aircraft_values: AviaryValues()):
         aircraft_values.set_val(
             Aircraft.Wing.FOLD_DIMENSIONAL_LOCATION_SPECIFIED, val=False)
 
-    if aircraft_values.get_val('verbosity').value >= 2:
+    if aircraft_values.get_val('settings:verbosity').value >= 2:
```

Comment:
Here and elsewhere in this file:
```suggestion
    if aircraft_values.get_val(Settings.VERBOSITY).value >= 2:
```